scripts/common.js

```diff
@@ -269,14 +314,14 @@ exports.populateFlexConfigPlaceholders = function populateFlexConfigPlaceholders
     serviceFunctionsDomain
      } = context
 
-    if(shell.test('-e', configFile)){
-      if(serviceFunctionsDomain){
-        shell.sed('-i', /<PLACEHOLDER_SERVERLESS_DOMAIN>/g, `${serviceFunctionsDomain}`, configFile);
-      }
-      if(serviceFunctionsDomain){
-        shell.sed('-i', /<PLACEHOLDER_SCHEDULE_MANAGER_DOMAIN>/g, `${scheduledFunctionsDomain}`, configFile);
-      }
+  if(shell.test('-e', configFile)){
+    if(serviceFunctionsDomain){
+      shell.sed('-i', /<PLACEHOLDER_SERVERLESS_DOMAIN>/g, `${serviceFunctionsDomain}`, configFile);
     }
+    if(serviceFunctionsDomain){
+      shell.sed('-i', /<PLACEHOLDER_SCHEDULE_MANAGER_DOMAIN>/g, `${scheduledFunctionsDomain}`, configFile);
+    }
```

Comment:
I know this was just a formatting change but while reading I noticed the variable being checked in the `if` statement is not correct here!